### DISCUSSION TITLE
Generate password

### DISCUSSION THREAD

**Discussion by anayrat** [sqlpage/SQLPage#496] [Q&A] [answered]
Hello,
I want my app to generate a strong password.
Basically, I can use exec function by calling "pwgen" utility. But that means allowing exec.
I wonder if there is another smarter solution ?
Else, I can try to create a SQL function to do that, but it won"t be trivial.
Thanks

**Comment by lovasoa** [comment, marked as answer]
hello! You should try [`sqlpage.random_string`](https://sql.ophir.dev/functions.sql?function=random_string#function) which generates strong random values.

**Comment by anayrat** [comment]
Indeed, I played with it and it seems to do the job. Maybe pwgen was overkill for my usage.
Thanks
